src/txn/txn.c

```diff
@@ -2422,6 +2422,11 @@ __wt_txn_global_shutdown(WT_SESSION_IMPL *session, const char **cfg)
               "[SHUTDOWN_INIT] performing shutdown rollback to stable, stable_timestamp=%s",
               __wt_timestamp_to_string(conn->txn_global.stable_timestamp, ts_string));
             WT_TRET(conn->rts->rollback_to_stable(session, cfg, true));
+
+            /* Log an error if rollback to stable fails here. */
+            if (ret != 0)
+                __wt_verbose(session, WT_VERB_RTS,
+                  "performing shutdown rollback to stable failed with code %d", ret);
```

Comment:
Would it be appropriate to use `__wt_strerror` here?

Reply:
I don't see why not! [1457a09](https://github.com/wiredtiger/wiredtiger/pull/8933/commits/1457a09053ba2de750c2016125d9401948051a2e)